docs/dev/new_check_howto.md

```diff
@@ -251,23 +249,23 @@ def test_service_check(aggregator):
     aggregator.assert_service_check('my_check.all_good', MyCheck.CRITICAL)
 ```
 
-Again, we could run all the tests but let's run only the integration tests for faster iterations:
+Run only the integration tests for faster iterations:
 
 ```
 tox -e integrations
 ```
 
-Our check is almost done, let's add the final touches.
+The check is almost done. Let's add the final touches.
 
 ## Final touches
 
-### Fill the README
+### Populate the README
 
-The `README.md` file as provided by our cookiecutter template has already the right format, just fill  out the relevant sections, in particular you're supposed to write something in place of the `[...]` strings you'll find in the file.
+The `README.md` file provided by our cookiecutter template already has the correct format. You must fill out the relevant sections - look for the ellipses `[...]`.
```

Comment:
@masci does cookiecutter provides the following format for the readme?: 

## Overview
## Setup
## Data Collected
## Troubleshooting
## Further Reading

Reply:
not exactly https://raw.githubusercontent.com/DataDog/cookiecutter-datadog-check/master/%7B%7Bcookiecutter.check_name%7D%7D/README.md

Reply:
super !